cfme/common/provider_views.py

```diff
@@ -523,10 +523,11 @@ class ContainerProviderAddViewUpdated(ContainerProviderAddView, ContainerProvide
     """
      Additional widgets for builds 5.9 and up
     """
-    COND_WIDGETS = ['prov_type', 'metrics_type', 'alerts_type']
+    COND_WIDGETS = ['prov_type', 'metrics_type', 'alerts_type', 'virtualization']
 
     metrics_type = BootstrapSelect(id='metrics_selection')
     alerts_type = BootstrapSelect(id='alerts_selection')
+    virtualization = BootstrapSelect(id='virtualization_selection')
```

Comment:
**Required**

Consistent naming with the other endpoint type selections, let's call this `virt_type` or `virtualization_type`.

Update the maps in the yaml accordingly